src/qililab/result/quantum_machines_results/qm_result.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Qilimanjaro Quantum Tech
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""QuantumMachinesResult class."""
+import numpy as np
+
+from qililab.constants import QMRESULT, RUNCARD
+from qililab.result.counts import Counts
+from qililab.result.result import Result
+from qililab.typings.enums import ResultName
+from qililab.utils.factory import Factory
+
+
+@Factory.register
+class QuantumMachinesResult(Result):
```

Comment:
Could you rename the file to `quantum_machines_result.py`?

Reply:
sure, done